slices/nodejs.yaml

```diff
@@ -0,0 +1,12 @@
+package: nodejs
+
+slices:
+  bins:
+    essential:
```

Comment:
I know it already comes from `libnode108`, but still, when it comes to dependencies I believe we should stay true to the deb, and in this case, `nodejs` needs `libc6` apparently

Reply:
Done

Reply:
Why `libgcc-s1_libs` too? It's not listed in the deb...

Reply:
> Why `libgcc-s1_libs` too? It's not listed in the deb...

It's not listed in the binary package, but if you do a `ldd` on the binary, you will see the binary executable requires `libgcc` runtime.

Reply:
yes but it is a nested dependency. `libnode108` should bring it in.

Reply:
Done